boat-bay-server/src/backbase/java/com/backbase/oss/boat/bay/web/views/dashboard/diff/DiffReportRenderer.java

```diff
@@ -130,6 +161,33 @@ private ContainerTag ol_changed(List<ChangedOperation> changedOperations) {
         return ol;
     }
 
+    private ContainerTag ol_tags(Map<String, Map<PathItem.HttpMethod, List<String>>> changedTags) {
+        ContainerTag ol = ol();
+        changedTags
+            .keySet()
+            .forEach(
+                pathUrl -> {
```

Comment:
create methods here as well

Reply:
Extracted li method and using forloop for better readability